prime-angular-frontend/src/app/modules/adjudication/shared/components/adjudication-container/adjudication-container.component.ts

```diff
@@ -231,6 +231,65 @@ export class AdjudicationContainerComponent extends AbstractComponent implements
       });
   }
 
+  public onDeclineEnrollee(enrolleeId: number) {
+    const data: DialogOptions = {
+      title: 'Decline Enrollee',
+      message: 'When declined the enrollee will not have access to PRIME and their Terms of Access revoked,'
+        + ' Are you sure you want to lock this enrollee ? ',
```

Comment:
You can get multiple lines using backticks and don't have to concatenate.  Also, might be worth updating "and their Terms of Access revoked" to "and their Terms of Access will be revoked"